### ISSUE TITLE
watch_videos endpoint doesn't seem to be supported

### ISSUE THREAD

**Issue by milogert** [jdf76/plugin.video.youtube#453]
Hello,

Hopefully this is the right place to leave this.

I have a site I use for ambient board game music (melodice.org). Right now they play Youtube videos on the actual site. Recently I contacted the developers to add the "Play on Youtube" link.

![screen shot 2018-10-10 at 7 36 41 am](https://user-images.githubusercontent.com/5378535/46736778-80d1e980-cc5f-11e8-82bc-0e214cdceda9.png)

(This example is located at https://melodice.org/playlist/gloomhaven-2017/)

The link works by using the `watch_videos` Youtube endpoint, providing a list of ids via the `video_ids` url parameter.

Would it be possible to support that in this plugin? On Youtube it ends up making an Untitled Playlist but I do not want to go through the hoops of trying to save the playlist publicly and then sending that to Kodi (also, I can't figure out how to do that on mobile).

Let me know if you need more information.

**Comment by goldmojo** [comment]
Hello @milogert , did you find a solution to play melodice.org playlists in Kodi ?
I am highly interested.
Best regards.

**Comment by milogert** [comment]
No, unfortunately I have been very busy (jeez, for the past year 😬 ).

In the end I ended up just linking my phone to my Alexa Dot and playing through that. I am still highly interested in this though.

I might give it some looking soon since my schedule is going to get shaken up a ton. Unfortunately I am not versed in developing and debugging Kodi plugins :(.

**Comment by goldmojo** [comment]
Hello @milogert, I finished creating a kodi addon dedicated to melodice.

You can test it from here : 
https://github.com/goldmojo/Kodi_addons/tree/master/_repo/plugin.video.melodice
(sources are available 2 directories upper)

The addon asks you for the name of your game and starts a youtube playlist playback.
Does it meet your gamer's expectations ?

**Comment by milogert** [comment]
Yeah! That's awesome. Do you have it listed on some package repo list? Or how can I add your repo to Kodi? I was able to install the package manually by zip on a local instance but I haven't tested on my actual media center yet.

**Comment by anxdpanic** [comment]
Thanks for sharing @goldmojo 
I'd suggest to update https://github.com/goldmojo/Kodi_addons/blob/master/plugin.video.melodice/addon.py#L93 to use `plugin://plugin.video.youtube/play/?video_id=` since `plugin://plugin.video.youtube/?action=play_video&videoid=` is deprecated and will be removed at some point

**Comment by goldmojo** [comment]
> Yeah! That's awesome. Do you have it listed on some package repo list? Or how can I add your repo to Kodi? I was able to install the package manually by zip on a local instance but I haven't tested on my actual media center yet.

Thank you for your comment, I created a brand new repository, you can find it here : https://github.com/goldmojo/repository/tree/master/zips/repository.goldmojo

**Comment by goldmojo** [comment]
> Thanks for sharing @goldmojo
> I'd suggest to update https://github.com/goldmojo/Kodi_addons/blob/master/plugin.video.melodice/addon.py#L93 to use `plugin://plugin.video.youtube/play/?video_id=` since `plugin://plugin.video.youtube/?action=play_video&videoid=` is deprecated and will be removed at some point

Thank you for this advice, I updated the code in your way :-)
I also used your repository as a template for mine, thanks.
